src/renderer/vt/XtermEngine.hpp

```diff
@@ -61,6 +63,8 @@ namespace Microsoft::Console::Render
         bool _previousLineWrapped;
         bool _usingUnderLine;
         bool _needToDisableCursor;
+        bool _lastCursorIsVisible = false;

```

Comment:
Shouldn't we initialize these in the constructor for consistency?

Reply:
agreed!